docs/api/migration-guides/qiskit-1.0.mdx

```diff
@@ -0,0 +1,659 @@
+
+
+<span id="for-users"></span>
+## For users
+
+You must start a new virtual environment to install Qiskit 1.0.
+It is very tricky and error-prone to upgrade an existing installation in-place to Qiskit 1.0.
+
+<Admonition type="note">
+The examples in this section use the `venv` module that is part of the Python standard library.
+If you use a different tool, such as `virtualenv` or `conda`, consult its documentation for help.
+
+For Linux and macOS commands, a bash-like syntax is used.
+PowerShell is used for Windows commands.
+</Admonition>
+
+<span id="creating-a-venv"></span>
+### Create the new environment
+
+1. Create a new virtual environment, using your preferred version of Python 3.8 or later. Use any path you choose in place of `/path/to/qiskit-1.0-venv`.
+
+        <OperatingSystemTabs>
+          <TabItem value="mac" label="macOS">
+             ```bash
+             python3 -m venv /path/to/qiskit-1.0-venv
+             ```
+          </TabItem>
+
+          <TabItem value="linux" label="Linux">
+             ```bash
+             python3 -m venv /path/to/qiskit-1.0-venv
+             ```
+          </TabItem>
+
+          <TabItem value="win" label="Windows">
+            ```powershell
+            python3 -m venv C:\path\to\qiskit-1.0-venv
+            ```
+          </TabItem>
+         </OperatingSystemTabs>
+
+<span id="activate"></span>
+2. Activate the environment.
+
+        <OperatingSystemTabs>
+          <TabItem value="mac" label="macOS">
+             ```bash
+             source /path/to/qiskit-1.0-venv/bin/activate
+             ```
+          </TabItem>
+
+          <TabItem value="linux" label="Linux">
+             ```bash
+             source /path/to/qiskit-1.0-venv/bin/activate
+             ```
+          </TabItem>
+
+          <TabItem value="win" label="Windows">
+            ```powershell
+            C:\path\to\qiskit-1.0-venv\Scripts\activate.ps1
+            ```
+          </TabItem>
+         </OperatingSystemTabs>
+
+3. Install packages as desired.
+   You should do this by using only one `pip install` command with all the dependencies on it.
+
+   ```bash
+   pip install 'qiskit>=1'
+   ```
+
+   You can optionally include additional packages by including them as arguments.  For example: 
+
+   ```bash
+   pip install 'qiskit>=1' jupyterlab pandas matplotlib
+   ```
+
+   Qiskit 1.0 includes breaking changes, so several packages are marked as not-yet-compatible with it.  Therefore, you might see errors from `pip` until new versions of those packages are released. Old versions of packages might also depend on the legacy `qiskit-terra` package.  Such packages might not return errors when running this command, but might raise an error when running `import qiskit`.
+
+   <Admonition type="caution">
+   Do not install packages that are incompatible with Qiskit 1.0 on this virtual environment.  If you need to use such packages, install them in a separate virtual environment with Qiskit 0.45 or 0.46.
```

Comment:
The biggest pitfall I see is installing some old dependency that depends on qiskit-terra explicitly. Should that be called out here as something to watch out for? Maybe with a link to the what changed section for more details?

Reply:
I've put an extra sentence just about the admonition explicitly saying that.